### PR TITLE
[#394] Fix bump version workflow

### PR DESCRIPTION
**Pull request by doannimble**
Close https://github.com/nimblehq/android-templates/issues/394

## What happened 👀

Replace the version name of the variable instead of changing it directly in bundle files

## Insight 📝

- Replace `ANDROID_VERSION_NAME` variable with the new value, by using `Perl` command

## Proof Of Work 📹

https://github.com/nimblehq/android-templates/pull/402

### CODE REVIEW

**Review by thiennguyen0196 — APPROVED** [review]
LGTM 👏

**Review by luongvo — APPROVED** [review]
👏

File: .github/workflows/bump_version.yml [hunk 1]
```diff
@@ -24,16 +24,14 @@ jobs:
           perl -i -pe 's/^(templateScriptVersion=(.*))$/"templateScriptVersion=${{ github.event.inputs.newVersion }}"/e' version.properties
 
       - name: Bump version XML
-        uses: chkfung/android-version-actions@v1.2.1
-        with:
-          gradlePath: sample-xml/app/build.gradle.kts
-          versionName: ${{ github.event.inputs.newVersion }}
+        id: set_version_xml
```
Comment (Tuubz) [inline .github/workflows/bump_version.yml:27]: @doannimble Just curious, what value does `id` bring? 🙏

**Review by Tuubz — APPROVED** [review]

**Comment by Tuubz** [comment]
@ryan-conway I have given you merge permissions (and get well soon) 💪

File: .github/workflows/bump_version.yml (hunk 1 above)
Comment (doannimble) [reply .github/workflows/bump_version.yml]: @Tuubz nothing, I've removed them. Thanks for your question 🙏

**Review by kaungkhantsoe — APPROVED** [review]

**Pull request merged**